pkgs/desktops/gnome-3/extensions/drop-down-terminal/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-drop-down-terminal-${version}";
```

```suggestion
  pname = "gnome-shell-extension-drop-down-terminal";
```

when attributes `pname` and `version` are present they're automatically used to make `name`
`name = "${pname}-${version}";`

---

Thanks, i didnt know about that!